framework/qlf/dashboard/bokeh/qaskyresid/main.py

```diff
@@ -54,10 +54,7 @@
 skyresid  = metrics['skyresid']
 snr       = metrics['snr']
 
-try:
-    skyresid = ast.literal_eval(skyresid)
-except:
-    sys.exit('Could not load skyresid metrics')
+skyresid = ast.literal_eval(skyresid)
```

Comment:
We changed the database so there is no need to use `ast.literal_eval` anymore anywhere.